server/util.py

```diff
@@ -134,6 +136,56 @@ def construct_slack_attachment(
 
     return attachment
 
+def get_circle_amount(rdo):
```

Comment:
We ended up slacking about this since that doesn't seem to work

![Screenshot 2024-04-04 at 9 18 03 AM](https://github.com/texastribune/donations/assets/2974713/4c9c778b-93be-49da-ad3d-0e182162e40b)